cocos/2d/renderer/render-data.ts

```diff
@@ -332,22 +339,26 @@ export class RenderData extends BaseRenderData {
         this.updateHash();
 
         if (JSB && this.multiOwner === false && this._renderDrawInfo) {
+            const renderDrawInfo = this._renderDrawInfo;
+            const chunk = this.chunk;
+            const meshBuffer = chunk.meshBuffer;
             // for sync vData and iData address to native
-            this._renderDrawInfo.setDrawInfoType(this._drawInfoType);
-            this._renderDrawInfo.setBufferId(this.chunk.bufferId);
-            this._renderDrawInfo.setVertexOffset(this.chunk.vertexOffset);
-            this._renderDrawInfo.setIndexOffset(this.chunk.meshBuffer.indexOffset);
-            this._renderDrawInfo.setVB(this.chunk.vb);
+            renderDrawInfo.setDrawInfoType(this._drawInfoType);
+            renderDrawInfo.setBufferId(chunk.bufferId);
+            renderDrawInfo.setVertexOffset(chunk.vertexOffset);
+            renderDrawInfo.setIndexOffset(meshBuffer.indexOffset);
+            renderDrawInfo.setVB(chunk.vb);
             // TODO: on TS 4.2, argument of type 'Readonly<Uint16Array>' is not assignable to parameter of type 'Uint16Array'.
-            this._renderDrawInfo.setIB(this.chunk.ib as Uint16Array);
-            this._renderDrawInfo.setVData(this.chunk.meshBuffer.vData.buffer);
-            this._renderDrawInfo.setIData(this.chunk.meshBuffer.iData.buffer);
-            this._renderDrawInfo.setVBCount(this._vc);
-            this._renderDrawInfo.setIBCount(this._ic);
+            renderDrawInfo.setIB(chunk.ib as Uint16Array);
+            renderDrawInfo.setVData(meshBuffer.vData.buffer);
+            renderDrawInfo.setIData(meshBuffer.iData.buffer);
+            renderDrawInfo.setVBCount(this._vc);
+            renderDrawInfo.setIBCount(this._ic);
         }
     }
 
-    protected setRenderDrawInfoAttributes (): void {
+    /** @mangle */
+    protected override setRenderDrawInfoAttributes (): void {
```

Comment:
Why can use mangle for protected function?

Reply:
Is it because it related with jsb?

Reply:
`RenderDrawInfo` relevant APIs are all used for `native (JSB)`. Non-native platforms don't need that.